.github/workflows/integration-tests.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
       DOCKER_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
       DOCKER_REGISTRY: quay.io
       REPO: quay.io/costoolkit/os2-ci
+      USE_FRAMEWORK_IMAGE: true # makes it use the $REPO-framework image for the chart
```

Comment:
just a note, we need to set this also in the release pipeline, this is going only to apply for integration tests otherwise

Reply:
currently just setting it for testing, I would like to discuss on monday if we should just create a ros-installer, bin only image instead and make it the default for our chart building here